database/sql/V072__auction_solution_orders.sql

```diff
@@ -0,0 +1,66 @@
+-- All auctions ran by autopilot
+CREATE TABLE competition_auctions (
+   id bigint PRIMARY KEY,
+   -- The block number at which the auction was created
+   block bigint NOT NULL,
+   -- The block number until which all winning solutions from a competition should be settled on-chain
+   deadline bigint NOT NULL,
+   -- Orders that were part of the auction
+   order_uids bytea[] NOT NULL,
+   -- The auction's native price tokens
+   price_tokens bytea[] NOT NULL,
+   -- The auction's price values
+   price_values numeric(78,0)[] NOT NULL,
+   -- A list of all surplus capturing JIT order owners that were part of the auction
+   surplus_capturing_jit_order_owners bytea[] NOT NULL
+);
+
+-- Table to store all proposed solutions for an auction, received from solvers during competition time.
+-- A single auction can have multiple solutions, and each solution can contain multiple order executions.
+-- This design allows for multiple solutions from a single solver
+CREATE TABLE proposed_solutions (
+   auction_id bigint NOT NULL,
+   -- Unique Id of the solution within the auction (generated by the autopilot)
```

Comment:
Why should this be generated by the autopilot?

Reply:
Why is that a useful thing to have? Also it would be weird to have **U** ids that are actually not unique.

Reply:
> Why is that (uniqueness only within an auction) a useful thing to have?

Another reason to be generated by autopilot is that we need aligned ids between `proposed_solutions` and `proposed_trade_executions`. And I am not sure there is a way to make this happen if we use batched inserts like we do now.

Reply:
Hmm, good point. Okay, then let's drop this discussion here.